test/t8n/t8n.cpp

```diff
@@ -128,6 +136,16 @@ int main(int argc, const char* argv[])
                 }
             }
 
+            if (block_reward.has_value())
+            {
+                if (*block_reward == 0)
+                    state.touch(block.coinbase);
+                else
+                    state.get_or_insert(block.coinbase).balance += *block_reward;
+            }
+            else if (rev <= EVMC_TANGERINE_WHISTLE)
```

Comment:
Don't we do the same in transaction execution?

Reply:
no, we are touching there here we are erasing.